lib/hexdocs/queue.ex

```diff
@@ -250,4 +256,61 @@ defmodule Hexdocs.Queue do
       )
     end)
   end
+
+  defp find_search_items(repository, package, version, files) do
+    maybe_json =
+      Enum.find_value(files, fn {path, content} ->
+        case Path.basename(path) do
+          "search_data-" <> _digest ->
+            "searchData=" <> json = content
+            json
+
+          "search_items-" <> _digest ->
+            "searchNodes=" <> json = content
+            json
+
+          _other ->
+            nil
+        end
+      end)
+
+    unless maybe_json do
+      Logger.error(
```

Comment:
Since packages that weren't built with hexdocs won't have this.

```suggestion
      Logger.info(
```